### PR TITLE
BETA: Register _discord.chaotic.is-a.dev

### PR DESCRIPTION
**Pull request by ryanabraham1**
Added `_discord.chaotic.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by iostpa — APPROVED** [review]

File: domains/_discord.chaotic.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ChaoticChaosTheGreat",
+        "email": "ryan.ryanabraham@gmail.com"
```
Comment (CuteDog5695) [inline domains/_discord.chaotic.json:4]: is the email valid? i dont think thats a valid email, try removing the ryan. from it (just noticed the email thing)

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/_discord.chaotic.json (hunk 1 above)
Comment (ChaoticChaosTheGreat) [reply domains/_discord.chaotic.json:4]: email is valid

Comment (CuteDog5695) [reply domains/_discord.chaotic.json:4]: send a email to that to verify, actually is valid soooo sorry about this, ill merge it now

**Review by iostpa — APPROVED** [review]
nvm email valid, sent a email to that and actually went through, sorry about this

**Pull request merged**